src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -32,6 +33,31 @@ export interface BreadcrumbProps {
   id?: string;
 }
 
+// Truncate breadcrumb text if beyond 60 characters then add ellipsis at the end.
+const breadcrumbText = (title) => {
+  if (title.length >= 60) {
+    return (
+      <Tooltip content={title} id={"tooltip-" + title}>
+        {truncateText(title, 60, true)}
+      </Tooltip>
+    );
+  } else {
+    return truncateText(title, 60, false);
+  }
+};
+
+const truncateText = (str, n, useWordBoundary) => {
```

Comment:
Both `breadcrumbText` and `truncateText` are doing something similar in the first check for the string. In `breadcrumbText`, if the title is more than 60 characters, then in `truncateText`, it also checks to see if it's less than 60 characters.

I think you can combine both functions into one function. If it's less than 60 characters, just return the text. If it's more than 60 characters, then you can shorten the text, add the ellipsis, and wrap the text in a `Tooltip`.